package.json

```diff
@@ -33,39 +28,27 @@
     "@babel/plugin-proposal-object-rest-spread": "7.8.3",
     "@babel/plugin-transform-react-jsx": "7.8.3",
```

Comment:
Can also be removed

Reply:
I also noticed that `eslint-utils` and `null-loader` could be removed, as well as `uuid`